omnibus/cookbooks/firezone/attributes/default.rb

```diff
@@ -381,17 +381,43 @@
 default['firezone']['robots_disallow'] = nil
 
 # ### Outbound Email Settings
-#
 # If none of these are set, the :sendmail delivery method will be used. Using
 # the sendmail delivery method requires that a working mail transfer agent
 # (usually set up with a relay host) be configured on this machine.
-#
-# SMTP will use the 'plain' authentication method.
-default['firezone']['from_email'] = nil
-default['firezone']['smtp_address'] = nil
-default['firezone']['smtp_password'] = nil
-default['firezone']['smtp_port'] = nil
-default['firezone']['smtp_user_name'] = nil
+default['firezone']['outbound_email']['from'] = nil
+default['firezone']['outbound_email']['provider'] = nil
+default['firezone']['outbound_email']['configs'] = {
```

Comment:
This is cool! So multiple providers can be used simultaneously?

Maybe we can clarify that here and link to the Swoosh Adapter docs so users can find more information if they need to? E.g. 

```
# Configure one or more providers below.
# See the Swoosh library documentation for more information on configuring adapters:
# https://github.com/swoosh/swoosh#adapters
```

Reply:
Yes technically you can use anything and everything that swoosh supports. As long as we have this light layer of string to adapter mapping. Currently I only included the most popular ones for simplicity. That also reminds me that I forgot to add gen_smtp, and move the config parsing code into mailer itself, so it can possibly be used at runtime.

However, we will need a solid story and design (UI/UX wise) about how to support multiple providers. Probably not to start with it as it'd be a niche requirement.

Reply:
Agreed, most users will just set up one provider for now. But it's nice to know it could support more -- maybe a useful feature for Enterprise users down the road!